tegra/tegra.c

```diff
@@ -212,13 +212,16 @@ int drm_tegra_bo_wrap(struct drm_tegra_bo **bop, struct drm_tegra *drm,
 
 	/* check handle table to see if BO is already open */
 	bo = lookup_bo(drm->handle_table, handle);
+
+	pthread_mutex_unlock(&table_lock);
```

Comment:
```
pthread_mutex_lock(&table_lock);
bo = lookup_bo(drm->handle_table, handle);
pthread_mutex_unlock(&table_lock);
```
...is starting to look like a pattern. Maybe we could make a `static struct drm_tegra_bo * lookup_bo_locked(void *table, uint32_t key)`-helper for it?